packages/react-components/components/WithPasteAware.tsx

```diff
@@ -9,17 +9,20 @@ interface PasteAwareProps {
   value: string
   shouldShowClipboard: (value: string) => boolean
   onChangeText: (text: string) => void
+  leftIcon?: React.ReactNode
 }
 
 export interface PasteAwareWrappedElementProps {
   isPasteIconVisible: boolean
   onPressPaste: () => void
   onChangeText: (text: string) => void
+  leftIcon?: React.ReactNode
 }
 
 interface PasteAwareState {
   isPasteIconVisible: boolean
   clipboardContent: string | null
+  leftIcon?: React.ReactNode
```

Comment:
The `PasteAwareState` doesn't need a `leftIcon`, it's just used in the props :)

Reply:
removed!